src/htdocs/model/record_model.php

```diff
@@ -114,7 +115,7 @@ public function getLastAvg($deviceId, $hours) {
         foreach (RecordModel::FIELDS as $f) {
             $fields[] = "AVG($f)";
         }
-        $fields = implode(",", $fields);
+        $fields = implode(',', $fields) . ', COUNT(*)';
```

Comment:
Let's do it this way:
```php
$fields[] = "COUNT(*) AS `cnt`"
$fields = implode(',', $fields);
```

Reply:
Sure, I will check if it works because I am not certain you can name the `COUNT(*)` select field (from my previous experiment it couldn't be named with `AS`).